src/main/java/uk/gov/ons/ctp/response/action/service/impl/ActionProcessingServiceImpl.java

```diff
@@ -252,6 +248,21 @@ private ActionRequest createActionRequest(final Action action, final ActionPlan
     actionRequest.setSurveyName(surveyDTO.getLongName());
     actionRequest.setSurveyRef(surveyDTO.getSurveyRef());
 
+    CommsTypeClassifiers classifiers = new CommsTypeClassifiers();
+    classifiers.setLegalBasis(surveyDTO.getLegalBasis());
+    classifiers.setRegion(businessUnitAttributes.getRegion());
```

Comment:
@pricem14pc I was thinking about the blank region code and wasn't sure if we wanted an internal representation to map these to?